spec/Section 7 -- Response.md

```diff
@@ -244,6 +264,161 @@ discouraged.
+iterator of the `films` field closes.
+
+```json example
+{
+  "hasNext": false
+}
+```
+
+#### Stream payload
+
+A stream payload is a map that may appear as an item in the `incremental` entry
+of a response. A stream payload is the result of an associated `@stream`
+directive in the operation. A stream payload must contain `items` and `path`
+entries and may contain `label`, `errors`, and `extensions` entries.
+
+##### Items
+
+The `items` entry in a stream payload is a list of results from the execution of
+the associated @stream directive. This output will be a list of the same type of
+the field with the associated `@stream` directive. If `items` is set to `null`,
+it indicates that an error has caused a `null` to bubble up to a field higher
+than the list field with the associated `@stream` directive.
+
+#### Defer payload
+
+A defer payload is a map that may appear as an item in the `incremental` entry
+of a response. A defer payload is the result of an associated `@defer` directive
+in the operation. A defer payload must contain `data` and `path` entries and may
+contain `label`, `errors`, and `extensions` entries.
+
+##### Data
+
+The `data` entry in a Defer payload will be of the type of a particular field in
+the GraphQL result. The adjacent `path` field will contain the path segments of
+the field this data is associated with. If `data` is set to `null`, it indicates
+that an error has caused a `null` to bubble up to a field higher than the field
+that contains the fragment with the associated `@defer` directive.
+
+#### Path
+
+A `path` field allows for the association to a particular field in a GraphQL
+result. This field should be a list of path segments starting at the root of the
+response and ending with the field to be associated with. Path segments that
+represent fields should be strings, and path segments that represent list
+indices should be 0-indexed integers. If the path is associated to an aliased
+field, the path should use the aliased name, since it represents a path in the
+response, not in the request.
+
+When the `path` field is present on a Stream payload, it indicates that the
+`items` field represents the partial result of the list field containing the
+corresponding `@stream` directive. All but the non-final path segments must
+refer to the location of the list field containing the corresponding `@stream`
+directive. The final segment of the path list must be a 0-indexed integer. This
+integer indicates that this result is set at a range, where the beginning of the
+range is at the index of this integer, and the length of the range is the length
+of the data.
+
+When the `path` field is present on a Defer payload, it indicates that the
+`data` field represents the result of the fragment containing the corresponding
+`@defer` directive. The path segments must point to the location of the result
+of the field containing the associated `@defer` directive.
```

Comment:
If the defer is at the root level then there is no field, in that case the path would be empty I think.
```
query { ...@defer {person(id: "cGVvcGxlOjE=") {name}}
```